.travis.yml

```diff
@@ -1,4 +1,9 @@
+sudo: required
 language: rust
+cache:
+  - cargo
+  - directories:
+      - "${HOME}/kcov/"
```

Comment:
This doesn't override the usual directories does it? Is it additive?

Reply:
As we spoke, it is an arbitrary dir cache for kcov